examples/log.js

```diff
@@ -18,26 +18,20 @@ const ipfs = new IPFS({
 ipfs.on('error', (err) => console.error(err))
 ipfs.on('ready', async () => {
   const keystore = Keystore.create(dataPath + '/keystore')
-  const identitySignerFn = (id, data) => {
-    const key = keystore.getKey(id)
-    return keystore.sign(key, data)
-  }
-  const access = new AccessController()
-
   let identityA, identityB, identityC
 
   try {
-    identityA = await IdentityProvider.createIdentity(keystore, 'identityA', identitySignerFn)
-    identityB = await IdentityProvider.createIdentity(keystore, 'identityB', identitySignerFn)
-    identityC = await IdentityProvider.createIdentity(keystore, 'identityC', identitySignerFn)
+    identityA = await IdentityProvider.createIdentity({ id: 'identityA', keystore })
+    identityB = await IdentityProvider.createIdentity({ id: 'identityB', keystore })
+    identityC = await IdentityProvider.createIdentity({ id: 'identityC', keystore })
```

Comment:
We could remove the `keystore` arg from all these which I believe would allow us to not require `Keystore` module separately. I suppose having it like it is now can also help showing the usage. Either way is fine, but I'd err on the side of simplifying where possible (ie. hide complexity).

Reply:
Agreed actually, I took out `access` and thought the same thing though with the case of `access`, the default `AccessController` doesn't let you  create any other kind or add write keys so thought it was limited in showing usage. Will remove `keystore` too to keep it clean and describe the options are in API.md